api.md

```
@@ -385,17 +386,27 @@ For other state preservation needs, save your data before calling this method (e
+Requires {@link PluginsConfig.CapacitorUpdater.allowPreview} to be `true`.
+When `appId` is provided, the preview session temporarily uses that app id
+for update checks until the user leaves the preview. Stats are skipped while
+the preview session is active.
 
 Use this before calling {@link set} for Expo Go-style preview flows.
 
+**Parameters**
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `options` | `StartPreviewSessionOptions | undefined` | Optional preview session options. |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)